jmix-reports/reports-flowui/src/main/resources/io/jmix/reportsflowui/view/reportwizard/report-wizard-creator-view.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <view xmlns="http://jmix.io/schema/flowui/view"
-      title="msg://reportWizard.title">
+      title=" ">
```

Comment:
If you don't specify a title, will it be equivalent to " "?

Reply:
it will make close button appear on the top left side of the window, with " " it is on the right side